vertexbufferlayout.cpp

```diff
@@ -0,0 +1,51 @@
+#include "vertexbufferlayout.h"
+
+#include <vector>
+
+#include "renderutils.h"
+
+VertexBufferLayout::VertexBufferLayout() :
+    mStride(0)
```

Comment:
Rather you should initialize this and mElements in the header as described above, like:

```
class BufferLayout
{
    int mStride {0};
    std::vector<Something> mElements {};
}
```